core/src/main/java/com/amplifyframework/devmenu/DevMenuEnvironmentFragment.java

```diff
@@ -38,6 +44,27 @@ public DevMenuEnvironmentFragment() { }
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.dev_menu_fragment_environment, container, false);
+        View envInfoView = inflater.inflate(R.layout.dev_menu_fragment_environment, container, false);
+        displayEnvironmentInfo((TextView) envInfoView.findViewById(R.id.env_info_text));
```

Comment:
I removed the TextView parameter from displayEnvironmentInfo() (setting the text of the TextView in onCreateView instead), but I set the return type for displayEnvironmentInfo() to a SpannableStringBuilder instead of a String in order to retain the styling applied to the Spannables appended to the SpannableStringBuilder.